src/main/kotlin/lotto/controller/LottoController.kt

```diff
@@ -1,39 +1,49 @@
 package lotto.controller
 
-import lotto.model.LottoMachine
-import lotto.model.PrizeNumbers
-import lotto.model.TicketsSeller
+import lotto.model.*
 import lotto.view.InputView
 import lotto.view.ResultView
 
 class LottoController {
     private val inputView = InputView()
     private val resultView = ResultView()
     private val ticketSeller = TicketsSeller()
+    private val ticketGenerator = TicketGenerator()
```

Comment:
Try moving the internal variables declared inside `LottoController` into the primary constructor and injecting them from the outside.
This will make your code more flexible and easier to modify.

This approach is called dependency injection, and it’s definitely worth looking into! 😊